include/nccl_ofi.h

```diff
@@ -247,6 +247,12 @@ struct nccl_net_ofi_device {
 	 */
 	char *name;
 
+	/*
+	 * node_uniue_id - combination of (16-23) bits of the guid and 56 bits of
```

Comment:
This is in the part of the code that isn't EFA specific.  Don't include EFA-specific details in this part of the code.  Most NICs have a UUID / GUID / whatever that is similar in use.